### PR TITLE
Force-read Secrets in Lookup, `0.3.1` Release

### LINKED ISSUE #26: Add option in configuration to force secret lookup and ignore cached secrets
### Proposed implementation:
There should be a check box when configuring the plugin to ignore cached secrets. Add this to the param data model, and then use the boolean to determine if `vault-clj`'s `read-secret` uses the `:force-read` option.

### Other details
Do a minor version bump and re-release

### PR DESCRIPTION
**Pull request by drassaby**
Resolves #26 

Checkboxes are weird so I went for a dropdown menu in the UI. Tested locally in docker and unit tests added.

## Minor version release
`0.3.1` up from `0.3.0`

## Other
Updated `CHANGELOG` to link to issues and PRs describing each change

### CODE REVIEW

File: src/amperity/gocd/secret/vault/plugin.clj [hunk 1]
```diff
@@ -218,14 +221,17 @@
...
+  [client gocd-lookup-keys force-read]
   (let [;; A map from Vault paths (as keywords) to their corresponding secret data (defaults to nil)
         paths-to-vals
         (->> gocd-lookup-keys
              (map #(first (str/split % #"#")))
              (into #{})
-             (map (fn [path] [(keyword path) (vault/read-secret client path {:not-found nil})]))
+             (map (fn [path]
+                    [(keyword path) (vault/read-secret client path {:not-found nil
+                                                                    :force-read force-read})]))
```
Comment (pyncc) [inline src/amperity/gocd/secret/vault/plugin.clj:234]: won't this be a string from configuration? I don't see when its parsed to a boolean.

Comment (drassaby) [reply src/amperity/gocd/secret/vault/plugin.clj:234]: Good eye 🇦🇺, fixed

Comment (drassaby) [reply src/amperity/gocd/secret/vault/plugin.clj:234]: Fixing tests now

**Review by pyncc — APPROVED** [review]
Nice, LGTM

**Pull request merged**